<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>384</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>549</x>
     <y>110</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="login_background">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>341</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_name">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_password">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>164</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="enter">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>127</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>登录</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="remember">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="automatic">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="timedata">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="regist">
   <property name="geometry">
    <rect>
     <x>549</x>
     <y>164</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>注册</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>162</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="baby">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>240</width>
     <height>240</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ferget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>208</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>11</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string> 找回密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close1">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>login_background</zorder>
  <zorder>close</zorder>
  <zorder>user_name</zorder>
  <zorder>user_password</zorder>
  <zorder>remember</zorder>
  <zorder>automatic</zorder>
  <zorder>timedata</zorder>
  <zorder>regist</zorder>
  <zorder>enter</zorder>
  <zorder>user1</zorder>
  <zorder>user2</zorder>
  <zorder>baby</zorder>
  <zorder>ferget</zorder>
  <zorder>close1</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
